doc : unity_specific.md

arthur740212 opened the pull request:
Information about Unity oddities

arthur740212 committed:
Update unity_specific.md

andrew-hirata-playeveryware commented on docs/unity_specific.md:
```diff
@@ -39,7 +39,9 @@ Similarly we can feed the `IntPtr` received from `LoadLibrary(string)` into `Fre
-### Reference to other documents on how and why we need a GFX plugin on windows
+### Why do we need a GFX plugin on Windows
+
+GFX plugin is required so that the EOS Overlay can render properly on Windows
```
* On line 14, it should read "has to be compatible with" instead of "has to compat with"
* Line 26 "rerun the plugin in without" should be "return the plugin without"
* The sentence on line 44 needs a period. 

As a general note, we may not need to talk about how we LoadLibrary / FreeLibrary.

arthur740212 committed:
doc : fix unity specific md

 ix grammar, typo, and  remove how to un/load libraries
add a little more info about gfx

scovertb commented on docs/unity_specific.md:
```diff
@@ -23,25 +23,14 @@ We take multiple factors into account, and decide on the versions that are most
-Similarly we can feed the `IntPtr` received from `LoadLibrary(string)` into `FreeLibrary(IntPtr)`, which should unload the library.  
-Hence allowing the sdk to modify its configurations.
-
+Therefore we unload and reload the dll for our users to rerun the plugin without needing to reboot Unity Editor. 
```
Inconsistent DLL capitalization, should be reboot _the_ Unity editor.

scovertb commented on docs/unity_specific.md:
```diff
@@ -23,25 +23,14 @@ We take multiple factors into account, and decide on the versions that are most
 
-GFX plugin is required so that the EOS Overlay can render properly on Windows
-
+GFX plugin is required so that the EOS Overlay can render properly on Windows.   
```
Remove all apostrophes from this paragraph

Raxnam reviewed (changes requested):
a few slight notes on lines it wont let me comment on

line 7: ... how we support EOS SKD ... should probably read ... how we support the EOS SDK ...
line 8: specify unity versions

scovertb commented on docs/unity_specific.md:
```diff
@@ -11,7 +11,7 @@ This document lists some Unity related oddities and how we support EOS SDK in ou
 ----------------------------------------------------------------------------------------
```
Line 1: Top header should probably be something like "Unity-specific plugin functionality"

andrew-hirata-playeveryware reviewed (commented):
Besides fixing the things that @scovertb mentioned, it looks good.

arthur740212 committed:
doc : fix

fix typos and grammar, added sdk versions

arthur740212 committed:
doc : correct unity and sdk version for public
